tutorials/cm006-exercise.Rmd

```diff
@@ -13,39 +18,154 @@ Change the file output to both html and md _documents_ (not notebook).
 
 1. Load the `gapminder` and `tidyverse` packages. Hint: `suppressPackageStartupMessages()`!
     - This loads `dplyr`, too.
+    
+```{r load packages, warning = F, message = F}
```

Comment:
Would prefer to use `FALSE` instead of `F`. Just better practice that I think we should demonstrate.

Reply:
Sounds good